source/_patterns/02-organisms/content-headers/content-header-profile.yaml

```diff
@@ -0,0 +1,54 @@
+assets:
+  css:
+    - content-header.css
+  js: []
+schema:
+  $schema: http://json-schema.org/draft-04/schema#
+  type: object
+  properties:
+    displayName:
+      type: string
+      minLength: 1
+    details:
+      type: object
+      properties:
+        affiliations:
+          type: array
+          items:
+            type: string
+            minLength: 1
+          minItems: 1
+        emailAddress:
+          type: string
+          minLength: 1
+      minProperties: 1
+    micsLinks:
+      type: array
+      items:
+        type: object
+        properties:
+          text:
+            type: string
+            minLength: 1
+          uri:
+            type: string
+            minLength: 1
+        required:
+          - text
+          - uri
+# is there a way of refactoring out the redundant property declarations between miscLinks and logoutLink?
```

Comment:
Since the `text`/`uri` combo existing in lots of places, we could have these stores elsewhere and use JSON References here to use them (https://spacetelescope.github.io/understanding-json-schema/structuring.html; on the API https://github.com/elifesciences/api-raml/tree/develop/src/misc is a bunch of reusable subschemas).